content/en/docs/features/control-plane/entity-tag.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'Entity-tag (ETAG)'
+description: 'What is ETAG?'
+docsOthersDisplay: true
+date: '2021-05-13'
+categories: [features]
+keywords: [twin, twin, cache, history]
+weight: 31
+---
+
+An entity-tag (ETAG) is used to identify changes of the resource that hapend over time. ETAG is generated by the resource server, in our case device, using various methods such as a version, checksum, hash or a timestamp. When the client retrieves an entity-tag, it should treat it as an opaque value without making any assumptions about its content or structure. In CoAP, the ETag is part of the [CoAP options](https://datatracker.ietf.org/doc/html/rfc7252#section-5.4) and is used to detect changes in resources. It is used optimizes the client processing time of the response from the device in case no change was detected - the ETAG wasn't changed from the last request.
+
+{{< note >}}
+For more information about ETAG, refer to the [RFC7252 Section 5.10.6](https://datatracker.ietf.org/doc/html/rfc7252#section-5.10.6).
+{{< /note >}}
+
+## IoTivity-lite
+
+### Definitions
+
+- **ETAG**: An ETAG is an 8-byte opaque value that represents the state of a resource. It is generated by the device and used to detect changes in resources.
+- **Global ETAG**: The Global ETAG is the highest ETAG value generated among all resources in the device. It is used to generate the next ETAG when a resource is created or modified.
+- **Collision**: A collision happens when a client has a cached resource with a specific ETAG value, but the device has a same ETAG associated with the different resource state. Collisions can occur when there is a lack of synchronization between the client and the device's time.
```

Comment:
The client's time is irrelevant. For the client, as you wrote, it's an opaque value.

Reply:
fixed